dataloader.py

```diff
@@ -0,0 +1,103 @@
+# PyTorch imports
+import torch
+from torch.utils.data import Dataset, DataLoader
+from torch.utils.data.sampler import SubsetRandomSampler
+
+# Other libraries for data manipulation and visualization
+import os
+import numpy as np
+
+
+class PadSequence:
+    def __call__(self, batch):
```

Comment:
Question: Why is this a class and not a function?

Reply:
Good point. I was looking for a solution to pad sequence and train LSTM in batch, and this was found on one of the discussion in PyTorch forum. Now when I think about it, this was pass to collate_fn in dataloader. Maybe it is better to use function instead.